styleguide/source/assets/scss/01-atoms/_table.scss

```diff
@@ -48,46 +48,51 @@ table.bt tbody tr:last-child {
   border-bottom: 1px solid $gray-50;
 }
 
-table.bt tbody td {
-  border: 0 solid $gray-20;
-  display: block;
-  vertical-align: top;
-
-  &:before {
-    content: attr(data-th) ":";
-    font-weight: 900;
-    text-transform: uppercase;
-    display: inline-block;
-
+table.bt {
+  &:not(.ama__table-agenda) {
+    tbody td {
+      &:before {
+        content: attr(data-th) ":";
+        font-weight: 900;
+        text-transform: uppercase;
+        display: inline-block;
+
+      }
+    }
   }
+  tbody td {
+    border: 0 solid $gray-20;
+    display: block;
+    vertical-align: top;
 
 
-  &:first-child + &:first-child {
-    border-top: 1px solid $gray-50;
-  }
+    &:first-child + &:first-child {
+      border-top: 1px solid $gray-50;
+    }
 
-  &:last-child {
-    @include gutter($padding-bottom-half...);
-    border-bottom: 1px solid $gray-50;
+    &:last-child {
+      @include gutter($padding-bottom-half...);
+      border-bottom: 1px solid $gray-50;
 
-  }
+    }
 
-  &:first-of-type:before,
-  &:first-of-type .bt-content {
-  }
+    &:first-of-type:before,
```

Comment:
can empty classes be removed?

Reply:
Yes. Thanks @designerneil! They are now removed.